pupil_src/shared_modules/video_export_launcher.py

```diff
@@ -77,12 +66,9 @@ def init_ui(self):
 
     def _update_ui(self):
         del self.menu.elements[:]
-        self.menu.append(
-            ui.Info_Text(
-                "Supply export video recording name. The export will be in the recording dir. If you give a path the export will end up there instead."
-            )
-        )
-        self.menu.append(ui.Text_Input("rec_name", self, label="Export name"))
+        # self.menu.append(ui.Info_Text('The export can be found in the exports folder, under the recording directory.'))
```

Comment:
I would suggest to delete the out-commented code because we can restore these lines from git history if we need them. Anyway, I'm not sure if I as a user would appreciate the info in the panel, the log output tends to vanish very quickly, any thoughts on that @papr ?

Reply:
Thanks for the comments @ckbaumann. My motivation behind removing most of the text, is that when you export, the full path to exported file is revealed. 
![screenshot from 2018-09-26 16-13-46](https://user-images.githubusercontent.com/6604563/46131464-f55a6080-c23b-11e8-8fd3-73a3afca69aa.png)

So it's redundant information, but only after the export is executed.

And I agree with you on removing the commented code.

Reply:
@ckbaumann fyi you can open the Log History plugin for more persistent log messages.

@ckbaumann @emredog 
I think this sentence is general enough and that we can keep it:
> The export can be found in the exports folder, under the recording directory.